### PR TITLE
Use root version number for projects

### PR DESCRIPTION
**Pull request by felix-oq**
Changes the setup so that the version from the root `package.json` is used for all published projects.

Also includes some fixes for broken scripts that I noticed along the way.

### CODE REVIEW

File: apps/interpreter/package.json [hunk 1]
```diff
@@ -18,5 +17,6 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org"
-  }
+  },
+  "version": "will be overridden by the root version"
```
Comment (georg-schwarz) [inline apps/interpreter/package.json:21]: Interesting idea for documentation :)
You sure it works?

Comment (georg-schwarz) [reply apps/interpreter/package.json:21]: Is inconsistent with other apps where the field was just removed.

File: tools/scripts/add-package-json-version.mjs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const [, , projectName] = process.argv;
+
+process.chdir(workspaceRoot);
+const rootPackageJson = parsePackageJson();
+
+process.chdir(getOutputPath(projectName));
+
+const packageJson = parsePackageJson();
+packageJson.version = rootPackageJson.version;
+writePackageJson(packageJson);
```
Comment (georg-schwarz) [inline tools/scripts/add-package-json-version.mjs:20]: My understanding: write to the `package.json` in the output directory, so not into the source one. Would we need to write into the package.lock there as well? Or does it not exist in the output dir?

File: apps/interpreter/package.json (hunk 1 above)
Comment (felix-oq) [reply apps/interpreter/package.json:21]: Maybe something like `the version field here is required for the interpreter, but the value will be overridden by the root version` would be better?

Comment (georg-schwarz) [reply apps/interpreter/package.json:21]: Aaah! Makes sense, thx! :)

File: tools/scripts/add-package-json-version.mjs (hunk 2 above)
Comment (felix-oq) [reply tools/scripts/add-package-json-version.mjs:20]: You're right with your understanding. `package-lock.json` files don't exist in the output dirs.

File: apps/interpreter/package.json (hunk 1 above)
Comment (felix-oq) [reply apps/interpreter/package.json:21]: You're welcome! Would it be better that way?

Comment (georg-schwarz) [reply apps/interpreter/package.json:21]: Would turn it around, but yes :)

**Review by georg-schwarz — APPROVED** [review]

**Pull request merged**